Package.swift

```diff
@@ -17,16 +17,19 @@ import PackageDescription
 
 let package = Package(
     name: "AEPAssurance",
-    platforms: [.iOS(.v11)],
+    platforms: [.iOS(.v12)],
     products: [
         .library(name: "AEPAssurance", targets: ["AEPAssurance"])
     ],
     dependencies: [
-        .package(url: "https://github.com/adobe/aepsdk-core-ios.git", .upToNextMajor(from: "4.0.0"))
+        // .package(url: "https://github.com/adobe/aepsdk-core-ios.git", .upToNextMajor(from: "5.0.0"))
+         .package(url: "https://github.com/adobe/aepsdk-core-ios.git", .branch("dev-v5.0.0"))
```

Comment:
you fixing this now before promotion to staging?

Reply:
The SPM script in CI will fail until we release Core 5.0.0. So i am planning on fixing all of these once Core is released.